### ISSUE TITLE
yum install kubelet-1.16.3 failed on aarch64

### ISSUE THREAD

**Issue by BedivereZero** [kubernetes/release#1366]
#### What happened:

`yum install kubelet-1.16.3` failed

#### What you expected to happen:

`yum install kubelet-1.16.3` succeed

#### How to reproduce it (as minimally and precisely as possible):

```console
$ sudo yum install kubelet-1.16.3
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
Resolving Dependencies
--> Running transaction check
---> Package kubelet.aarch64 0:1.16.3-0 will be installed
--> Processing Dependency: kubernetes-cni >= 0.7.5 for package: kubelet-1.16.3-0.aarch64
Package kubernetes-cni is obsoleted by kubelet, but obsoleting package does not provide for requirements
--> Processing Dependency: socat for package: kubelet-1.16.3-0.aarch64
--> Processing Dependency: conntrack for package: kubelet-1.16.3-0.aarch64
--> Running transaction check
---> Package conntrack-tools.aarch64 0:1.4.4-7.el7 will be installed
--> Processing Dependency: libnetfilter_cttimeout.so.1(LIBNETFILTER_CTTIMEOUT_1.1)(64bit) for package: conntrack-tools-1.4.4-7.el7.aarch64
--> Processing Dependency: libnetfilter_cttimeout.so.1(LIBNETFILTER_CTTIMEOUT_1.0)(64bit) for package: conntrack-tools-1.4.4-7.el7.aarch64
--> Processing Dependency: libnetfilter_cthelper.so.0(LIBNETFILTER_CTHELPER_1.0)(64bit) for package: conntrack-tools-1.4.4-7.el7.aarch64
--> Processing Dependency: libnetfilter_queue.so.1()(64bit) for package: conntrack-tools-1.4.4-7.el7.aarch64
--> Processing Dependency: libnetfilter_cttimeout.so.1()(64bit) for package: conntrack-tools-1.4.4-7.el7.aarch64
--> Processing Dependency: libnetfilter_cthelper.so.0()(64bit) for package: conntrack-tools-1.4.4-7.el7.aarch64
---> Package kubelet.aarch64 0:1.16.3-0 will be installed
--> Processing Dependency: kubernetes-cni >= 0.7.5 for package: kubelet-1.16.3-0.aarch64
Package kubernetes-cni is obsoleted by kubelet, but obsoleting package does not provide for requirements
---> Package socat.aarch64 0:1.7.3.2-2.el7 will be installed
--> Running transaction check
---> Package kubelet.aarch64 0:1.16.3-0 will be installed
--> Processing Dependency: kubernetes-cni >= 0.7.5 for package: kubelet-1.16.3-0.aarch64
Package kubernetes-cni is obsoleted by kubelet, but obsoleting package does not provide for requirements
---> Package libnetfilter_cthelper.aarch64 0:1.0.0-11.el7 will be installed
---> Package libnetfilter_cttimeout.aarch64 0:1.0.0-7.el7 will be installed
---> Package libnetfilter_queue.aarch64 0:1.0.2-2.el7 will be installed
--> Finished Dependency Resolution
Error: Package: kubelet-1.16.3-0.aarch64 (kubernetes)
           Requires: kubernetes-cni >= 0.7.5
           Available: kubernetes-cni-0.5.1-0.aarch64 (kubernetes)
               kubernetes-cni = 0.5.1-0
           Available: kubernetes-cni-0.5.1-1.aarch64 (kubernetes)
               kubernetes-cni = 0.5.1-1
           Available: kubernetes-cni-0.6.0-0.aarch64 (kubernetes)
               kubernetes-cni = 0.6.0-0
           Available: kubernetes-cni-0.7.5-0.aarch64 (kubernetes)
               kubernetes-cni = 0.7.5-0
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

#### Environment:

- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):  CentOS Linux release 7.7.1908 (AltArch)
- Kernel (e.g. `uname -a`): Linux 192-168-139-51 4.18.0-80.7.2.el7.aarch64 #1 SMP Thu Sep 12 16:13:20 UTC 2019 aarch64 aarch64 aarch64 GNU/Linux
- Others:

**Comment by justaugustus** [comment]
Closed as duplicate of https://github.com/kubernetes/kubernetes/issues/92242. Please follow that issue for updates.
/close
